test/lib/creation.go

```diff
@@ -50,12 +53,39 @@ var coreAPIVersion = corev1.SchemeGroupVersion.Version
 var rbacAPIGroup = rbacv1.SchemeGroupVersion.Group
 var rbacAPIVersion = rbacv1.SchemeGroupVersion.Version
 
+// This is a workaround for https://github.com/knative/pkg/issues/1509
+// Because tests currently fail immediately on any creation failure, this
+// is problematic. On the reconcilers it's not an issue because they recover,
+// but tests need this retry.
+//
+// https://github.com/knative/eventing/issues/3681
+func isWebhookError(err error) bool {
+	return strings.Contains(err.Error(), "eventing-webhook.knative-eventing")
+}
+
+func (c *Client) RetryWebhookErrors(updater func(int) error) error {
```

Comment:
Can we keep this private?

Reply:
I can, but if this works, I was thinking of hoisting it to pkg so, then it's just a replace here? Makes sense?

Reply:
@vaikas sounds good to me